src/pages/partners/_premium_agent.module.scss

```diff
@@ -0,0 +1,50 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+
+  .wrapper {
+    width: 100%;
+    max-inline-size: 56rem;
+  }
+
+  .array_wrapper {
+    display: grid;
+    gap: 2.5rem;
+    inline-size: 296px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(5, 1fr);
+    margin-block-start: 2.5rem;
+
+    .country {
+      display: flex;
+      align-items: center;
+      gap: 1rem;
+    }
+
+    @media (min-width: 600px ) {
+      inline-size: 54rem;
+      gap: 4rem;
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: repeat(3, 1fr);
```

Comment:
Can we reach our needs without grid? Later Michio will move some special things to grid and will teach us how to work with it
And will provide to us something like `GridBox` so now let's use `FlexBox` 🙏

Reply:
I am using FlexBox.Item in this current situation but without grid and scss I can't set the specific rows and columns. I think until we get something like GridBox we can't do otherwise if we want a specific amount of rows and columns

Reply:
On the call we discussed how to reach our needs with `flex-basis` 😄